src/main/java/com/sugarcrm/candybean/automation/Candybean.java

```diff
@@ -187,11 +195,22 @@ public static synchronized Candybean getInstance() throws CandybeanException {
 	public AutomationInterface getInterface() throws Exception {
 		throw new Exception("There are no non-webdriver automation interfaces currently defined.");
 	}
-	
+
+	public static String getDefaultConfigFile() throws CandybeanException{
```

Comment:
`getDefaultConfigFile()` should probably be private as the calling method `getDefaultConfiguration()` is private and most importantly, it doesn't give user much value to use the config file in `File` and the user shouldn't do that anyway.